common/src/main/java/org/apache/rocketmq/common/BrokerConfig.java

```diff
@@ -392,11 +392,14 @@ public boolean isEnable() {
     private long channelExpiredTimeout = 1000 * 120;
     private long subscriptionExpiredTimeout = 1000 * 60 * 10;
 
+    private int batchDispatchRequestThreadPoolNums = 16;
```

Comment:
Why not put this config to MessageStoreConfig?

Reply:
> Why not put this config to MessageStoreConfig?

a small detail, niubility